src/lighteval/models/nanotron_model.py

```diff
@@ -1133,7 +1133,7 @@ def greedy_until(
             else:
                 # Longest context in the current split is the first item (since we sort reversed)
                 context_enc = dataset[0][1].tokenized_context
-                max_gen = max(item[1].generation_size for item in dataset)
+                max_gen = max(item.generation_size for item in dataset)
```

Comment:
was this creating an error and is it linked to another issue or PR ?

Reply:
I think this came from another branch: https://github.com/huggingface/lighteval/pull/344